apps/authentication-portal/src/main/webapp/basicauth.jsp

```diff
@@ -185,7 +185,13 @@
     }
 %>
 
-<% if (Boolean.parseBoolean(loginFailed) && !errorCode.equals(IdentityCoreConstants.USER_ACCOUNT_NOT_CONFIRMED_ERROR_CODE)) { %>
+<% if (StringUtils.equals(request.getParameter("errorCode"), IdentityCoreConstants.USER_ACCOUNT_LOCKED_ERROR_CODE) &&
+        StringUtils.equals(request.getParameter("remainingAttempts"), "0") ) { %>
+<div class="ui visible negative message" id="error-msg" data-testid="login-page-error-message">
+    <%=AuthenticationEndpointUtil.i18n(resourceBundle, "error.user.account.locked.incorrect.login.attempts")%>
+</div>
+<% } else if (Boolean.parseBoolean(loginFailed) &&
+        !errorCode.equals(IdentityCoreConstants.USER_ACCOUNT_NOT_CONFIRMED_ERROR_CODE)) { %>
```

Comment:
better to equal the error code to the constant as in `!IdentityCoreConstants.USER_ACCOUNT_NOT_CONFIRMED_ERROR_CODE.equals(errorCode))`

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/2458